dark/assets/css/styles.css

```diff
@@ -1056,39 +1058,6 @@ html {
   margin-bottom: 0;
 }
 
-#contact h3 {
-  text-align: center;
-  color: white;
-  font-size: 60px;
-  position: relative;
-  bottom: 25px;
-  font-family: impact;
-  z-index: 99;
-  filter: blur(1px);
-}
-
-#contact h3::before {
-  content: "";
-  position: absolute;
-  display: block;
-  width: 130px;
-  height: 2px;
-  background: rgb(168, 168, 168);
-  bottom: -25px;
-  left: calc(50% - 60px);
-}
-
-#contact h3::after {
-  content: "";
-  position: absolute;
-  display: block;
-  width: 50px;
-  height: 4px;
-  background: #0563bb;
-  bottom: -26px;
-  left: calc(50% - 20px);
-}
-
 .formContainer {

```

Comment:
these also from line 1061 - 1094

Reply:
I have pushed the changes. This is what it looks like.

![Screenshot 2024-10-08 at 16-58-41 YOSHITHA RATHNAYAKE](https://github.com/user-attachments/assets/e9cae082-a27b-4f17-a0d3-56b3001ba98c)